composer/algorithms/utils/augmentation_common.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 MosaicML. All Rights Reserved.
+
+from typing import Type, TypeVar, cast
+
+import torch
+from PIL.Image import Image as PillowImage
+from torchvision import transforms
+
+_InputImgT = TypeVar("_InputImgT", torch.Tensor, PillowImage)
+_OutputImgT = TypeVar("_OutputImgT", torch.Tensor, PillowImage)
+
+
+def image_as_type(image: _InputImgT, need_type: Type[_OutputImgT]) -> _OutputImgT:
+    """Creates a copy of an image-like object with a different type.
+
+    Args:
+        image: a tensor or PIL image. Batches of images in NCHW format are
```

Comment:
docstrings say batch is supported but then below there is a check?

Reply:
Clarified the docstring. The check makes it fail if we ask to convert a batch of >1 image to a PIL image. We could have it convert to a collection of PIL images, but randaug and augmix don't support that (seemed like too muchto add in one PR), so we just fail fast instead.